db/connectDb.py

```diff
@@ -0,0 +1,60 @@
+from main import POPULATION_THRESHHOLD, CityInfo
+from dotenv import dotenv_values
+import psycopg2.extras
+
+
+config = dotenv_values('.env')
+POSTGRESQL_PASSWORD = config.get('POSTGRESQL_PASSWORD')
+POSTGRES_HOST = config.get('POSTGRES_HOST')
+POSTGRESQL_USERNAME = config.get('POSTGRESQL_USERNAME')
+
+CONN = psycopg2.connect(
+    host=POSTGRES_HOST,
+    database='postgres',
+    user=POSTGRESQL_USERNAME,
+    password=POSTGRESQL_PASSWORD
+)
+
+def getCitiesByPopulation(populationThreshold):
+    cur = CONN.cursor(cursor_factory=psycopg2.extras.DictCursor)
+
+    cur.execute(''' 
+                    SELECT city, country, lng, lat, city_id FROM worldwidecities
+                    WHERE population >= %s
+                    ORDER BY random() LIMIT 1;  
+                ''', [populationThreshold])
+
+    city = cur.fetchall()
+    for col in city:
+        city, country, lng, lat, cityId = col['city'], col['country'], col['lng'], col['lat'], col['city_id']
```

Comment:
This is very personal so you can leave as-is if you like this, but

I cannot stand this style of formatting which is funny considering I'm a destructuring nut. Leave it, but know that some folks have beef with this formatting